pallets/attestation/src/lib.rs

```diff
@@ -191,10 +193,22 @@ impl<T: Trait> Module<T> {
 	}
 }
 
+#[derive(Encode, Decode)]
+pub struct Attestation<T: Trait> {
+	// hash of the CTYPE used for this attestation
+	ctype_hash: T::Hash,
+	// the account which executed the attestation
+	owner: T::AccountId,
```

Comment:
I just noticed in the sdk that owner is very misleading. Who owns an attestation? That's a philosophical question that we all might answer differently. The owner in our case is the attester?
 
```suggestion
	// the account which executed the attestation
	attester: T::AccountId,
```

Reply:
That is true. Will change.

Reply:
Fixed in 9b0fe51